libraries/AP_Scripting/applets/rover-quicktune.lua

```diff
@@ -26,6 +23,7 @@ local MAV_SEVERITY = {EMERGENCY=0, ALERT=1, CRITICAL=2, ERROR=3, WARNING=4, NOTI
 
 local PARAM_TABLE_KEY = 12
```

Comment:
change table key?

Reply:
Sure, updated to 15 which also doesn't conflict with other scripts. txs